model_analyzer/triton/client/client_factory.py

```diff
@@ -23,29 +23,33 @@ class TritonClientFactory:
     """
 
     @staticmethod
-    def create_grpc_client(server_url):
+    def create_grpc_client(server_url, ssl_options={}):
         """
         Parameters
         ----------
         server_url : str
             The url for Triton server's GRPC endpoint
+        ssl_options : dict
+            Dictionary of SSL options for gRPC python client
```

Comment:
Don't we send the entire dict here? Inclusive of https? As there is a single `get_ssl_options` function. Would it make more sense to have `get_https_ssl_options` and `get_grpc_ssl_options`?

Reply:
At the moment, I think we leave as is.
Can you create a ticket to fix this later?

Reply:
Refactored to two functions 👍